Makefile

```
@@ -115,33 +118,41 @@ govulncheck:
 
 release-dev:
 	$Q @docker run -it --rm --privileged -e CGO_ENABLED=1 \
+		-e GORELEASER_KEY=${GORELEASER_KEY} \
+		-e IS_PRERELEASE=true \
 		--entrypoint /bin/bash \
-		-v /var/run/docker.sock:/var/run/docker.sock \
-		-v `pwd`:/go/src/$(PKG) \
+		-v $(DOCKER_SOCK):/var/run/docker.sock:Z \
+		-v `pwd`:/go/src/$(PKG):Z \
 		-w /go/src/$(PKG) \
-		ghcr.io/goreleaser/goreleaser-cross:${GOLANG_CROSS_VERSION}
+		ghcr.io/goreleaser/goreleaser-cross-pro:${GOLANG_CROSS_VERSION}
 
 release-dry-run:
 	$Q @docker run --rm --privileged -e CGO_ENABLED=1 \
+		-e GORELEASER_KEY=${GORELEASER_KEY} \
+		-e GPG_PRIVATE_KEY_FILE=/dev/null \
+		-e IS_PRERELEASE=true \
 		--entrypoint /go/src/$(PKG)/docker/build/entrypoint.sh \
-		-v /var/run/docker.sock:/var/run/docker.sock \
-		-v `pwd`:/go/src/$(PKG) \
+		-v $(DOCKER_SOCK):/var/run/docker.sock:Z \
+		-v `pwd`:/go/src/$(PKG):Z \
 		-w /go/src/$(PKG) \
-		ghcr.io/goreleaser/goreleaser-cross:${GOLANG_CROSS_VERSION} \
-		--clean --skip=validate --skip=publish
+		ghcr.io/goreleaser/goreleaser-cross-pro:${GOLANG_CROSS_VERSION} \
+		--clean --skip=validate --skip=sign --prepare
```

Although is the same, for consistency:
```suggestion
		release --clean --skip=validate --skip=sign --prepare
```